test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -130,6 +140,37 @@ def test_qdel_nagle_perf(self):
         self.logger.info(
             "qdel performance after setting manager: " + str(qdel_perf2))
 
-        # Check that the two timings are pretty close to each other
-        self.assertTrue((qdel_perf2 - qdel_perf) < 5,
-                        "qdel performance differs too much!")
+    @timeout(600)
+    def test_qsub_perf(self):
+        """
+        Test that qsub performance have improved when run
+        with -f option
+        """
+
+        # Restart server again
+        self.server.restart()
+
+        # Submit a job for 500 times and timeit
+        cmd = os.path.join(
+              self.server.client_conf['PBS_EXEC'],
+              'bin',
+              'qsub -- /bin/true >/dev/null')
+        start_time = timeit.default_timer()
```

Comment:
I like the use of timeit.  After reading up about it, it sounds like the perfect way to do these timing tests.  Now both tests use different methods of timing though.  Maybe modify the previous test to use timeit?